docker-compose.yml

```diff
@@ -3,41 +3,23 @@ services:
   grafana:
     image: grafana/grafana:latest
     volumes:
-      - ./grafana/provisioning/dashboards/:/etc/grafana/provisioning/dashboards/
-      - ./grafana/provisioning/datasources/:/etc/grafana/provisioning/datasources/
-      - ./grafana/dashboards:/var/lib/grafana/dashboards
-      - ./grafana/grafana.ini:/etc/grafana/grafana.ini
+    - ./grafana/provisioning/dashboards:/etc/grafana/provisioning/dashboards
+    - ./grafana/provisioning/datasources:/etc/grafana/provisioning/datasources
+    - ./grafana/dashboards:/var/lib/grafana/dashboards
+    - ./grafana/grafana.ini:/etc/grafana/grafana.ini
     environment:
       GF_SECURITY_ADMIN_PASSWORD: codesmith
       GF_AUTH_ANONYMOUS_ENABLED: 'true'
       GF_PATHS_CONFIG: /etc/grafana/grafana.ini
     ports:
-      - '3000:3000'
-    depends_on:
-      - prometheus
-      - prometheus3
-  prometheus:
-    image: prom/prometheus:latest
-    volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml:ro
-      - ./kafka_controller_alerts.yml:/etc/prometheus/kafka_controller_alerts.yml:ro
-      - prometheus_data:/prometheus
-    ports:
-      - '9090:9090'
+    - '3000:3000'
   alertmanager:
     image: prom/alertmanager:latest
     ports:
-      - '9094:9094'
+    - '6093:6093'
```

Comment:
why 6093?

Reply:
Just needed to set it to something that wouldn't create port conflicts as we add new instances of prometheus. It was listening on port 9093 before so prometheus #4 would always wind up bailing.

Reply:
Also why 430AM